incubator/kubirds-datadog-reactor/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: kubirds-datadog-reactor
+description: Parametrable Datadog Reactor for Kubirds
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```

Comment:
Is datadog in version `1.16.0`?
Is the docker image used in the reactor provided by this chart in version `1.16.0`?

Reply:
I don't know which version to use

For example, this chart permit to use both v1/v2 api
Datadog agent use the v7

Help please ^^'